src/API/WorkController.ts

```diff
@@ -33,8 +35,17 @@ export class WorkController {
 
   async getByFilters(worksFilters: WorksFilters = {}): Promise<any> {
     this.logger.trace({ method: 'getByFilters', worksFilters }, 'Getting Work by Filters from DB')
+    const { offset }: { offset?: string } = omitUndefined(worksFilters)
+    const skip = parseInt(offset, 10) || 0
     const definedFilters = getDefinedFilters(worksFilters)
-    return this.collection.find(definedFilters, { fields: { _id: false } }).toArray()
+    return this.collection
+      .find(definedFilters, { fields: { _id: false } })
```

Comment:
`fields` is deprecated, use `projection` instead. See http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#find